tests/functional/lib/receptorcontrol/receptorcontrol.go

```diff
@@ -235,14 +253,20 @@ func (r *ReceptorControl) WorkRelease(workID string) error {
 	return nil
 }
 
-func (r *ReceptorControl) getWorkStatus(workID string) (map[string]interface{}, error) {
-	_, err := r.WriteStr(fmt.Sprintf("work status %s\n", workID))
+// GetWorkStatus returns JSON of status file for a given unitID
+func (r *ReceptorControl) GetWorkStatus(unitID string) (workceptor.StatusFileData, error) {
```

Comment:
I think we probably want to return a `*workceptor.StatusFileData` here

Reply:
done!